homeassistant/components/wled/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "domain": "wled",
+    "name": "WLED",
+    "config_flow": true,
+    "documentation": "https://www.home-assistant.io/integrations/wled",
+    "requirements": [
+        "wled==0.1.0"
+    ],
+    "dependencies": [],
+    "zeroconf": [
+        "_wled._tcp.local."
+    ],
+    "codeowners": [
+        "@frenck"
+    ]
+}
```

Comment:
Missing newline.

Reply:
The file is formatted using Prettier in VSCode, which is used across the codebase, including our frontend.

So as far as I am concerned, this is correct.

Reply:
Hmm... I found a GitHub issue @ Prettier that may relate to this. I'll double-check!

Reply:
Even worse, somehow my VSCode didn't even use Prettier as the default formatter. Now it indeed has it. Will update all JSON files.

Thanks!